ui/src/app/app-routing.tsx

```diff
@@ -246,4 +98,16 @@ export class AppRouting extends ReactWrapperBase {
     AnalyticsTracker.Registration.SignIn();
     this.signInService.signIn();
   }
+
+  onSignOut(): void {
+    this.signInService.isSignedIn$.subscribe(signedIn => {
```

Comment:
on sign out, we're subscribing to sign in? I don't think this is correct

Reply:
you're right, this should be named better, and I should call it in the useEffect with the rest of the subscriptions